model_compression_toolkit/hardware_models/get_model.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import importlib
+
+from model_compression_toolkit.common.hardware_representation import FrameworkHardwareModel
+
+found_tf = importlib.util.find_spec("tensorflow") is not None and importlib.util.find_spec(
+    "tensorflow_model_optimization") is not None
+found_torch = importlib.util.find_spec("torch") is not None
+
+tf_models_dict = {}
+torch_models_dict = {}
+
+if found_tf:
+    from model_compression_toolkit.hardware_models.keras_hardware_model.keras_default import KERAS_DEFAULT_MODEL
+    from model_compression_toolkit.hardware_models.keras_hardware_model.keras_tflite import KERAS_TFLITE_MODEL
+
+    tf_models_dict = {'default': KERAS_DEFAULT_MODEL,
+                      'tflite': KERAS_TFLITE_MODEL}
+if found_torch:
+    from model_compression_toolkit.hardware_models.pytorch_hardware_model.pytorch_default import PYTORCH_DEFAULT_MODEL
+    from model_compression_toolkit.hardware_models.pytorch_hardware_model.pytorch_qnnpack import PYTORCH_QNNPACK_MODEL
+
+    torch_models_dict = {'default': PYTORCH_DEFAULT_MODEL,
+                         'qnnpack': PYTORCH_QNNPACK_MODEL}
```

Comment:
Add pytorch tflite and qnnpack tensorflow

Reply:
In progress...